src/components/alert/alert.html

```diff
@@ -4,10 +4,69 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/src/components/alert/alert.css">
     <title>Alert</title>
+    <script src="https://kit.fontawesome.com/a578b3d91b.js" crossorigin="anonymous"></script>
 </head>
 <body>
+ <!------------------------ Header ----------------------------------->
+ <header>
+    <div class="header-container">
+        <div class="heading-header">
+        <h1> Cali-Aesthetics</h1>
+        </div>
+        <div class="header-links">
+        <a class="link-primary">Home</a>
+        <a class="link-primary">Docs</a>
+        </div>
+    </div>
+  </header>
 
+
+<!-------------------------Side Bar ----------------------------->
+<aside>
+<h3>Components</h3>
+<a href="/src/components/avatar/avatar.html">Avatar</a>
+<a class="active-tab" href="/src/components/alert/alert.html">Alert</a>
+<a href="/src/components/badge/badge.html">Badge</a>
+<a href="/src/components/button/button.html">Button</a>
+<a href="#">Card</a>
+<a href="#">Input</a>
+<a href="#">Form</a>
+<a href="#">Snackbar</a>
+</aside>
+
+
+
+<main>
+
+ <!-- ------------ component : alert ---------------- -->
+
+<div class=" main-container padding-md">
+    <div class="headline">
+    <h1>Alert</h1>
+    <div class="description">
+        Alerts component provides important message to attract users attention without any interrupting.
+    </div>
+    </div>
+
+<section class="background-white">
+    <h3>Exampes of Alert</h3>
+    <div class="section-description ">
+        Alerts are available in four types i.e Primary Alert, Successful Alert, Danger Alert and Warning Alert. To use this component, you have to add <span class="highlighted">alert</span>, according to your need you can use these classes <span class="highlighted">primary-alert</span>, <span class="highlighted">successfull-alert</span>, <span class="highlighted">danger-alert</span>, <span class="highlighted">warning-alert</span>.
+    </div>
+    <div class="container-column">
+        <div class="alert primary-alert"><i class="fas fa-solid fa-circle-info"></i>Primary Alert</div>
+        <div class="alert successfull-alert"><i class="fas fa-solid fa-circle-check"></i> Successfull Alert</div>
+        <div class="alert warning-alert"><i class="fas fa-solid fa-triangle-exclamation"></i> Warning Alert</div>
+        <div class="alert danger-alert"><i class="fas fa-solid fa-skull-crossbones"></i> Danger Alert</div>
+    </div>
+</section>
+</div>
+<i class="fa-light fa-circle-info"></i>
```

Comment:
what is this icon doing?
I am not able to see this icon in UI.

Reply:
deleted that icon